daemon/runtime_unix_test.go

```diff
@@ -242,11 +244,11 @@ func TestGetRuntime(t *testing.T) {
 	assert.Assert(t, ok, "stock runtime could not be found (test needs to be updated)")
 	stockRuntime.Features = nil
 
-	configdOpts := *stockRuntime.Opts.(*v2runcoptions.Options)
+	configdOpts := proto.Clone(stockRuntime.Opts.(*v2runcoptions.Options)).(*v2runcoptions.Options)
```

Comment:
This commit probably as well together with the containerd update (or does it compile without?)